cdap-ui/app/cdap/components/Replicator/Detail/index.tsx

```diff
@@ -0,0 +1,190 @@
+import { Map, fromJS } from 'immutable';
+import If from 'components/If';
+import { Link, Redirect } from 'react-router-dom';
+import moment from 'moment';
+import Status from 'components/Status';
+
+const styles = (theme): StyleRules => {
+  return {
+    root: {
+      padding: '25px 40px',
+    },
+    buttonContainer: {
+      '& > *': {
+        marginRight: '15px',
+      },
+    },
+    config: {
+      border: `1px solid ${theme.palette.grey[300]}`,
+      borderRadius: '4px',
+      '& > pre': {
+        wordBreak: 'break-word',
+        whiteSpace: 'pre-wrap',
+        padding: '15px',
+      },
+    },
+  };
+};
+
+interface IDetailProps extends WithStyles<typeof styles> {
+  match: {
+    params: {
+      replicatorId: string;
+    };
+  };
+}
+
+const DetailView: React.FC<IDetailProps> = ({ classes, match }) => {
+  const [replicator, setReplicator] = React.useState(Map());
+  const [error, setError] = React.useState();
+  const [status, setStatus] = React.useState();
+  const [redirect, setRedirect] = React.useState(false);
+
+  function getBaseParams() {
+    return {
+      namespace: getCurrentNamespace(),
+      appName: match.params.replicatorId,
+    };
+  }
+
+  React.useEffect(() => {
+    MyReplicatorApi.getReplicator(getBaseParams()).subscribe((app) => {
+      let config;
+
+      try {
+        config = JSON.parse(app.configuration);
+      } catch (e) {
+        setError(e);
+      }
+
+      setReplicator(fromJS(config));
+    });
+
+    MyReplicatorApi.pollStatus(getBaseParams()).subscribe((res) => {
+      setStatus(res.status);
+    });
+  }, []);
+
+  function start() {
+    const params = {
+      ...getBaseParams(),
+      action: 'start',
+    };
+
+    MyReplicatorApi.action(params).subscribe(fetchStatus, (err) => {
+      setError(err);
+    });
+  }
+
+  function stop() {
+    const params = {
+      ...getBaseParams(),
+      action: 'stop',
+    };
+
+    MyReplicatorApi.action(params).subscribe(fetchStatus, (err) => {
+      setError(err);
+    });
+  }
+
+  function deleteReplicator() {
+    MyReplicatorApi.delete(getBaseParams()).subscribe(
+      () => {
+        setRedirect(true);
+      },
+      (err) => {
+        setError(err);
+      }
+    );
+  }
+
+  function fetchStatus() {
+    MyReplicatorApi.getStatus(getBaseParams()).subscribe((res) => {
+      setStatus(res.status);
+    });
+  }
+
+  const listViewLink = `/ns/${getCurrentNamespace()}/replicator`;
+
+  if (redirect) {
+    return <Redirect to={listViewLink} />;
+  }
+
+  const startTime = moment()
+    .subtract(7, 'days')
+    .format('X');
+  let logUrl = `/v3/namespaces/${getCurrentNamespace()}/apps/${
+    match.params.replicatorId
+  }/workers/DeltaWorker/logs`;
+
+  logUrl = `${logUrl}?start=${startTime}`;
+  logUrl = `/downloadLogs?type=raw&backendPath=${encodeURIComponent(logUrl)}`;
+
+  return (
+    <div className={classes.root}>
+      <div>
+        <Link to={listViewLink}>Back to List View</Link>
```

Comment:
i18n for this text (and button text as well)?

Reply:
detail view in this PR is just a placeholder so that people can start/stop/delete the replicator